client/src/components/Markdown/directives.yml

```diff
@@ -0,0 +1,111 @@
+---
+history_dataset_display:
+  side_panel_name: Dataset
```

Comment:
Because that isn't how you would name them or give them descriptions. The side panel blurs them together is a presentation-specific way that I am not comfortable calling the name or description. Take "Current Time" "as text" for instance - I think Sam broke it up that way for the formatting - the description of that directive is definitely not "as text" and the abstract name is the directive name I think ``generate_time``. This YAML file is encoding some presentation specific stuff and I think that is fine.